sql/without_telemetry.sql

```diff
@@ -2,6 +2,7 @@
 -- Please see the included NOTICE for copyright information and
 -- LICENSE-APACHE for a copy of the license.
 
-CREATE OR REPLACE FUNCTION @extschema@.get_telemetry_report() RETURNS jsonb
-    AS '@MODULE_PATHNAME@', 'ts_telemetry_get_report_jsonb'
-	LANGUAGE C STABLE PARALLEL SAFE;
+-- We delete the telemetry job since this is a non-telemetry build.
+DELETE FROM _timescaledb_config.bgw_job WHERE id = 1;
```

Comment:
I'm a bit confused here, why do we need this DELETE since we'll run the `with_telemetry.sql` (where INSERT resides) just if we set the USE_TELEMETRY flag? Or is this for backward compatibility?

Reply:
This is to be able to handle the upgrade scenario where we upgrade from a server with a telemetry job to one without.